core/src/navigation/services/navigation.js

```diff
@@ -1,12 +1,20 @@
-import { LuigiConfig } from '../../services/config';
-import { getConfigValueFromObjectAsync } from '../../utilities/async-helpers';
+import {
+  LuigiConfig
+} from '../../services/config';
+import {
+  getConfigValueFromObjectAsync
+} from '../../utilities/async-helpers';
 
 const isNodeAccessPermitted = (
   nodeToCheckPermissionFor,
   parentNode,
   currentContext
 ) => {
-  if (!isLoggedIn()) return false;
```

Comment:
I see this already 3 times in our codebase, we could create a helper in LuigiConfig class and reuse it instead of duplicating code. [Here my proposal](https://github.com/kyma-project/luigi/files/2599011/luigi-222-login-fix.zip).

Reply:
Pushed by myself as requested from pr creator